cmd/signozschemamigrator/schema_migrator/traces_migrations.go

```diff
@@ -1,3 +1,360 @@
 package schemamigrator
 
 var TracesMigrations = []SchemaMigrationRecord{}
+
+// move them to TracesMigrations once it's ready to deploy
+var TracesMigrations_Staging = []SchemaMigrationRecord{
+	{
+		MigrationID: 1000,
+		UpItems: []Operation{
+			CreateTableOperation{
+				Database: "signoz_traces",
+				Table:    "signoz_index_v3",
+				Columns: []Column{
+					{Name: "ts_bucket_start", Type: ColumnTypeUInt64, Codec: "DoubleDelta, LZ4"},
```

Comment:
Is there any reason to use `LZ4` over `ZSTD`?

Reply:
not sure if there is any specific reason for choosing LZ4.

Reply:
In general, ZSTD performs well over LZ4 unless we have evidence to show that LZ4 is better.

Reply:
I tested it, seems there is a tiny advantage of LZ4 over zstd

For timestamp columns LZ4 is performing better

┌─database──────┬─table─┬─column─┬─compressed─┬─uncompressed─┬─compr_ratio─┬───rows_cnt─┬─avg_row_size─┐
│ signoz_traces │ test  │ tszstd │ 17.90 MiB  │ 15.48 GiB    │      885.54 │ 2077663885 │            8 │
│ signoz_traces │ test  │ tslz   │ 17.66 MiB  │ 15.48 GiB    │      897.67 │ 2077663885 │            8 │
└───────────────┴───────┴────────┴────────────┴──────────────┴─────────────┴────────────┴──────────────┘

The difference is not that much but the CPU usage will be less according to docs. Also we were using LZ4 for timestamp columns previously.